### PR TITLE
Do not uglify the JS bundle when executing `npm run debug`

### PR DESCRIPTION
**Pull request by msaelices**
See issue: https://github.com/nativescript-vue/vue-cli-template/issues/62

### CODE REVIEW

File: template/webpack.config.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ require('./prepare')();
 // Generate platform-specific webpack configuration
 const config = (platform, launchArgs) => {
 
+  const command = launchArgs.split(' ')[0];
+  const isDevelopment = command === 'debug'
```
Comment (arzke) [inline template/webpack.config.js]: Isn't _isDevelopment_ confusing? You are checking if we're running debug, not really development. If you want to cover development as well, then maybe you can do:
`const isDevelopment = command !== "build";`

What do you think?

**Comment by msaelices** [comment]
I think I agree partially. The name is confusing but the condition is ok as the only moment you do not need the uglify is when you are debugging, right? So maybe isDebug is a more proper name

**Comment by msaelices** [comment]
@brundozer Variable name changed here: https://github.com/nativescript-vue/vue-cli-template/pull/63/commits/26ec0416e75bd12d89382ea0c9444ca2d538bb45

**Comment by rigor789** [comment]
@msaelices does this work in `watch` as well? I usually don't need the debugger, and I just run `npm run watch:android/ios` where it would still make sense to not uglify to get better line numbers in errors.

The proposed check by @brundozer might be a better fit: `command !== "build"` which would only uglify in builds.

**Comment by msaelices** [comment]
@rigor789 done: https://github.com/nativescript-vue/vue-cli-template/pull/63/commits/9ef76f8efa7dd9b9c363700ed940af9a934dba07

Could you PTAL?

**Comment by rigor789** [comment]
LGTM! 👍

**Pull request merged**